### PR TITLE
Attempt at separating out creating list vs contributing a list #266

### PR DESCRIPTION
**Pull request by RichardLitt**
What do you think?

### CODE REVIEW

**Comment by sindresorhus** [comment]
Good idea, but I would prefer for it to be a separate file. I don't want to junk up the readme with that.

**Comment by RichardLitt** [comment]
What should the filed be called? `Creating A List.md`?

**Comment by sindresorhus** [comment]
`create-list.md` and should be linked to from contributing.md and readme.md

**Comment by RichardLitt** [comment]
How about that?

File: create-list.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Creating Your Own List
+
+- Read [the awesome manifesto](https://github.com/sindresorhus/awesome/blob/master/awesome.md) and ensure your list complies.
+- Wait at least a couple of weeks after creating a list before submitting to this list to give it a chance to mature.
+- Search this list before making a new one, as yours may be a duplicate. If it is, try and contribute to the best one instead of making your own.
```
Comment (sindresorhus) [inline create-list.md]: Add an entry about reading the contribution guidelines and linkify it.

File: readme.md [hunk 2]
```diff
@@ -33,7 +34,6 @@ If you want to contribute, please read the [contribution guidelines](contributin
 - [Security](#security)
 - [Miscellaneous](#miscellaneous)
 
-
```
Comment (sindresorhus) [inline readme.md]: ?

File: contributing.md [hunk 3]
```diff
@@ -1,12 +1,11 @@
 # Contribution Guidelines
 
+## Adding to this list
```
Comment (sindresorhus) [inline contributing.md]: `## Contributing to this list`

File: contributing.md [hunk 4]
```diff
@@ -18,6 +17,10 @@ Please ensure your pull request adheres to the following guidelines:
 
 Thank you for your suggestions!
 
+## Creating your own awesome list
+
+To create your own list, check out the [create list instructions](create-list.md).
```
Comment (sindresorhus) [inline contributing.md]:
```
To create your own list, check out the [instructions](create-list.md).
```

File: create-list.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Creating Your Own List
+
+- Read [the awesome manifesto](https://github.com/sindresorhus/awesome/blob/master/awesome.md) and ensure your list complies.
```
Comment (sindresorhus) [inline create-list.md]:
```
- Read the [awesome manifesto](https://github.com/sindresorhus/awesome/blob/master/awesome.md) and ensure your list complies.
```

**Comment by RichardLitt** [comment]
Fixed.

**Comment by sindresorhus** [comment]
image

**Pull request merged**